README.md

```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch as Tracking Store in Mlflow. To use this plugin you need a running instance of Elasticsearch 6.8.
```

Comment:
I would rephrase this, the tracking service is MLflow, and elastic search is a backend you can use with this plugin. Also does it need to be Elasticsearch 6.8 ? Doesn't it work with 6.X ?

Reply:
Haven't tested it but it should work